src/lib/eco-router/api.ts

```diff
@@ -296,6 +307,15 @@ export async function getExactOut(
             tradeType: TradeType.EXACT_OUTPUT,
           })
         }
+        if (platform.name === RoutablePlatform.OPENOCEAN.name) {
+          return SwaprV3Trade.getQuote({
+            quoteCurrency: currencyIn,
+            amount: currencyAmountOut,
+            maximumSlippage,
+            recipient: receiver,
+            tradeType: TradeType.EXACT_OUTPUT,
+          })
+        }
```

Comment:
This doesn't look right 😄

Reply:
Ahhh it was a test and you passed (?
Lol sorry, addressing it right now